src/test/ui/parser/issue-99625-enum-struct-mutually-exclusive.stderr

```diff
@@ -0,0 +1,8 @@
+error: `enum` and `struct` are mutually exclusive
+  --> $DIR/issue-99625-enum-struct-mutually-exclusive.rs:3:5
+   |
+LL | pub enum struct Range {
+   |     ^^^^^^^^^^^ help: replace `enum struct` with `enum`: `enum`
```

Comment:
Why `enum`: `enum`?

Reply:
I think "help: replace `enum struct` with: `enum`" or "help: replace with: `enum`" is much better